packages/vscode-extension/src/webview/components/DimensionsBox.tsx

```diff
@@ -0,0 +1,123 @@
+import { InspectData } from "../../common/Project";
+import { DeviceProperties } from "../utilities/consts";
+import "./DimensionsBox.css";
+
+const BOX_HEIGHT_FRACTION = 0.033;
+const FONT_SIZE_FRACTION = 0.6;
```

Comment:
is there e method ou used to come to those values? if so please describe it in the comment here

Reply:
They are 100% arbitrary and I have chosen them such that I like how the box looks like 😄